### DISCUSSION TITLE
What is "Open files as new task" feature?

### DISCUSSION THREAD

**Discussion by shuvashish76** [TeamAmaze/AmazeFileManager#2766] [Q&A] [answered]
Settings> Miscellaneous> Open files as new task
![20210902_000046](https://user-images.githubusercontent.com/48226312/131724714-dc313cc0-9e5c-426e-bdd0-453c596e29a7.jpg)

It seems the current option is not the feature I expected... #2744

1st Q&A discussion. Welcome everyone to Amaze Discussions :)
🙏🏿

**Comment by EmmanuelMess** [comment, marked as answer]
This allows you to open text files as if they were "new apps" so that they show as recent apps, but currently doesn't work, this is an issue.
